test/sumMainSecundaryDiagonal.test.js

```diff
@@ -0,0 +1,40 @@
+var expect = require("chai").expect;
+
+const mainfunction = require("../sumMainSecundaryDiagonal");
+
+describe("Sum of main and secundary Diagonals", function () {
```

Comment:
change the message to something like: "Add sumatory of primary and secondary diagonals"